content/2020/2020-07/2020-07-10-pulsar-weekly.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "2020-07-10-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2020-07-04 ~ 2020-07-10"
+date: 2020-07-04 ~ 2020-07-10
+description: "This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on."
+id: "2020-07-10-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-07-04 ~ 2020-07-10
+
+This is the Pulsar community weekly update for 2020-07-04 ~ 2020-07-10, with updates on Pulsar authentication, connectors, Functions, geo replication, and so on.
+
+### Notable Feature
+
+- [Broker] Use CGroup CPU usage when present.
+
+    https://github.com/apache/pulsar/pull/7475
+    
+- [C++ Client] Add support for oauth2 auth.
+
+    https://github.com/apache/pulsar/pull/7467
+    
+- [Function] Differentiate authorization between source/sink/function operations.
+
+    https://github.com/apache/pulsar/pull/7466
+    
+- [Function] Allow function rebalance to be run periodically.
+
+    https://github.com/apache/pulsar/pull/7449
+    
+### Notable Bug Fix
+
+- [Gep Replication] Handle `NoSuchElemetnException` when update geo replication policy.
```

Comment:
```suggestion
- [Gep Replication] Handle `NoSuchElemetnException` when updating geo replication policy.
```

Reply:
shall we refine the title or keep it consistent with the PR title?

Reply:
Think this description is clear enough.